src/systemtest/resources/features/ArtifactFlow.feature

```diff
@@ -11,36 +11,35 @@ Feature: Artifact system test
     And a jenkins job "TCSTCFJob" from "src/systemtest/resources/JenkinsShellScripts/TCSTCFScript.txt" is created with parameters: EVENT_ID,
     And a jenkins job "CLMJob" from "src/systemtest/resources/JenkinsShellScripts/CLMScript.txt" is created with parameters: EVENT_ID,
     And a jenkins job "ArtPJob" from "src/systemtest/resources/JenkinsShellScripts/ArtPScript.txt" is created with parameters: EVENT_ID,
-    And a jenkins job "FlowCompleteJob" from "src/systemtest/resources/JenkinsShellScripts/FlowCompleteScript.txt" is created with parameters: EVENT_ID,
+    And a jenkins job "FlowCompleteJob" from "src/systemtest/resources/JenkinsShellScripts/FlowCompleteScript.txt" is created with parameters: ,
 
     #####Add CLMSubscription to EI#####
-    Given subscription object "CLMSubscription" is created which will trigger "CLMJob"
+    Given subscription object "CLMSubscription" is created which will trigger "CLMJob" with parameters
```

Comment:
Is parameters added in this step? It looks like the parameters is added in next step..

Reply:
When the subscription is created, it needs to know if it will carry parameters or not since the notificationmeta will be different e.g "/build" or "/buildWithParameters". I should maybe change the name to, "which will carry parameters"?

Reply:
I would rather see that notification meta is set on it´s own line. But since the gurka is created with a lot of magic behind then I presume it must be ok cause it would require too much change in already merged gurka-code.

Reply:
Ye i think as little details as possible should be written in cucumber code. If we can hide implementation details like that behind normal language it's great. It would be too cluttered to set all those fields here.